csharp/test/Ice/proxy/AllTests.cs

```diff
@@ -569,8 +569,7 @@ public static Test.IMyClassPrx allTests(global::Test.TestHelper helper)
             test(proxyProps["Test.LocatorCacheTimeout"].Equals("100"));
             test(proxyProps["Test.InvocationTimeout"].Equals("1234"));
 
-            test(proxyProps["Test.Locator"].Equals(
-                        "locator -t -p ice1 -e " + Encoding.Latest.ToString()));
+            test(proxyProps["Test.Locator"].Equals($"locator -t -p ice1 -e {Encoding.V2_0}"));
```

Comment:
This is the only significant change to the tests (Encoding.Latest is currently 1.1).
I think the previous code was incorrect: when you Clone a proxy and set both the locator and encoding as part of this Clone, the specified encoding should override the locator proxy's encoding. (Here we Clone and specify encoding 2.0 and pass a locator proxy with the 1.1 encoding).